Sandbox fixes

shriharirathod opened the pull request.

Commit:
DP-1934: Custom port for REST

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

Commit:
DP-1908: Adding memory size and disks

Adding memory size and disks as config args

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

Commit:
DP-1908: Parsing new args

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

Commit:
Update config value to avoid overcommit

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

sphansalkar commented on src/hyc_asd_mgr.py:
```diff
@@ -34,10 +34,24 @@
+MWC_MEMORY_MARKER = "MWC_MEMORY"
+PWQ_MEMORY_MARKER = "PWQ_MEMORY"
+
+config_high = { "max-write-cache" : 536870912,
```
Can we have some formula which can make it generic. So that if compute config is changed from HA side, then we need not make changes in this code again

Commit:
Fixed issue while parsing disks

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

Commit:
Calculating memory dynamically

    Calculating the memory requirements for aerospike
    dynamically keeping in account excess available memory.

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

Commit:
Updating sizing calculations

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

Commit:
Fixed typo

Signed-off-by: Shrihari Rathod <shrihari.rathod@primaryio.com>

sphansalkar reviewed (approved):
LGTM
